### ISSUE TITLE
start/stop listening for all keys

### ISSUE THREAD

**Issue by karanba** [chieffancypants/angular-hotkeys#206]
there need a way to say stop/continue for listening. For example if there is an modal already on screen I do not want to enable other shortcuts do action..

**Comment by maksimr** [comment]
Agree, It will be nice to have something like isolate shortcuts, which will disable other shortcuts

**Comment by mkoczorowski** [comment]
you can unbind and then rebind keys form the scope

**Comment by juliadou** [comment]
+1. It will be very nice to disable hotkeys when a dialog is open so the hotkeys won't trigger callbacks which affect the UI under the dialog backdrop.

**Comment by newzen** [comment]
+1. have same problem with modal popup, some key used are used too in backdrop.

**Comment by vir2bille** [comment]
There are methods pause() and unpause() in API

**Comment by sohail85** [comment]
Is there a way to overcome this issue? 
since my hotkeys bounded to scope and controllers scope will not be destroyed since there are not route change while opening a stateless dialog, it will still run the hotkeys call back when the dialog is open and separate set of hotkeys been register on dialog (cheetshet display hotkeys of dialog as well as the page on top of which the dialog is currently open).
expected behavior is to only have dialog related hotkeys and page related hotkeys need to be unregistered or disabled
